tests/lib/pcm_mix/src/main.c

```diff
@@ -23,6 +23,9 @@ void test_mix_with_nothing(void)
 	int16_t sample_a[] = { 0, 1, -1 };
 	int16_t sample_r[] = { 0, 1, -1 };
 
+	/*TODO: injected failure */
+	assert_equal(1, 0, "Failing on purpose");
```

Comment:
Remove before merging?

Reply:
Yes. I will do that as soon that I verify that this fails.

Reply:
This still needs to be removed :)